bundles/org.openhab.binding.mffan/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="mffan" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Modern Forms Fan Binding</name>
+	<description>This is the binding for "Modern Forms", and "WAC Lighting" smart ceiling fans.</description>
+
```

Comment:
Can you add connection tag and maybe country and discovery ? (last two might be more difficult)